### PR TITLE
Simple refactorization, added project scripts for django project

### PR DESCRIPTION
**Pull request by matllubos**
installation

### CODE REVIEW

File: developers_chamber/bin/pydev.py [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+
 import logging.config
 
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+# Load configuration
+load_dotenv(dotenv_path=Path.home() / '.pydev' / 'pydev.conf')
+load_dotenv(dotenv_path=Path.cwd() / '.pydev' / 'pydev.conf')
```
Comment (matllubos) [inline developers_chamber/bin/pydev.py]: config is loaded from current dir bud from the home dir too

Comment (matllubos) [reply developers_chamber/bin/pydev.py]: current dir means project dir

File: developers_chamber/bin/pydev.py [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
 from developers_chamber.scripts.version import *
-from dotenv import load_dotenv
 
-load_dotenv(dotenv_path='./config/pydev.conf')
+
+# Import external scripts
+for base_path in (Path.home(), Path.cwd()):
+    if (base_path / '.pydev' / 'scripts').exists():
+        sys.path.append(base_path / '.pydev')
+        import scripts
```
Comment (matllubos) [inline developers_chamber/bin/pydev.py]: imported extra scripts for the concrete project

File: developers_chamber/docker_utils.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import click
 
+
```
Comment (matllubos) [inline developers_chamber/docker_utils.py]: pep8

File: developers_chamber/project_utils.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    if var_dir:
+        shutil.rmtree(var_dir, ignore_errors=True)
+
+    for _, _, host_dir in containers_dir_to_copy:
+        os.makedirs(Path.cwd() /host_dir)
+
+    compose_build(project_name, compose_files)
+
+    for container_name, container_dir, host_dir in containers_dir_to_copy:
```
Comment (matllubos) [inline developers_chamber/project_utils.py]: the reason of this are site-packages.

File: developers_chamber/scripts/__init__.py [hunk 5]
```diff
@@ -1,6 +1,27 @@
...
+import json
 import click
 
 
 @click.group()
 def cli():
     pass
+
+
+for k, v in json.loads(os.environ.get('ALIASES', '{}')).items():
```
Comment (matllubos) [inline developers_chamber/scripts/__init__.py]: dynamic commands from alias

File: developers_chamber/scripts/docker.py [hunk 6]
```diff
@@ -14,24 +14,24 @@ def docker():
 
 
 @docker.command()
-@click.option('-u', '--username', help='registry username', type=str, cls=RequiredIfNotEmpty,
+@click.option('--username', '-u', help='registry username', type=str, cls=RequiredIfNotEmpty,
               required_if_empty='password')
-@click.option('-p', '--password', help='registry password', type=str, cls=RequiredIfNotEmpty,
+@click.option('--password', '-p', help='registry password', type=str, cls=RequiredIfNotEmpty,
```
Comment (matllubos) [inline developers_chamber/scripts/docker.py]: just refactorization

File: developers_chamber/scripts/ecs.py [hunk 7]
```diff
@@ -6,12 +6,13 @@
 from developers_chamber.ecs_utils import stop_service as stop_service_func
 from developers_chamber.ecs_utils import start_service as start_service_func
 from developers_chamber.ecs_utils import register_new_task_definition as register_new_task_definition_func
-from developers_chamber.ecs_utils import update_service_to_latest_task_definition as update_service_to_latest_task_definition_func
+from developers_chamber.ecs_utils import update_service_to_latest_task_definition as \
```
Comment (matllubos) [inline developers_chamber/scripts/ecs.py]: pep8 and added shortcuts

File: developers_chamber/click/options.py [hunk 8]
```diff
@@ -21,3 +21,35 @@ def handle_parse_result(self, ctx, opts, args):
...
+            self.fail(
+                'Invalid value "{}" format must be "DOCKER_CONTAINER_NAME:CONTAINER_DIRECTORY:HOST_DIRECTORY"'.format(
+                    value
+                ),
+                param,
+                ctx,
+            )
+
+
+class ContaineCommandType(click.ParamType):
```
Comment (radimsuckr) [inline developers_chamber/click/options.py]: Missing `r` in `Container`.

File: developers_chamber/bin/pydev.py (hunk 1 above)
Comment (matllubos) [reply developers_chamber/bin/pydev.py]: we will ad it later now it is not required to use home config

Comment (matllubos) [reply developers_chamber/bin/pydev.py]: I won't solve mac os problems now..

**Pull request merged**